scripts/Makefile

```diff
@@ -0,0 +1,17 @@
+GO ?= go
+
+magic-gen: clean magic.h
+	$(GO) run magicgen.go magic.h ../magic/magic.go
+	rm -f magic.h
+
+magic.h:
+	curl -s https://raw.githubusercontent.com/checkpoint-restore/criu/criu-dev/criu/include/magic.h -o magic.h
+
+test:
+	@./magicgen-test.sh
+	@rm -f input.h output.go expected.go
```

Comment:
If the test fails it would be helpful to be able to look at the `input.h`, `output.go`, `expected.go` files. We could add them in `.gitignore` instead of removing them.

Reply:
The `exit 1` in the test script ensures that the files are not deleted if the test fails. They are removed only when the test runs successfully.

Reply:
You are right, the files are removed only when the test runs successfully.

It would still be useful to add them in `.gitignore`. Otherwise they appears as untracked files and may be accidentally be added in a git commit.
```
$ git status
...
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	expected.go
	input.h
	output.go
```